tests/test_file.py

```diff
@@ -62,6 +64,27 @@ def test_that_a_local_temporary_path_is_created_when_staging_a_remote_file(test_
     assert remote_file.local_path == staged_path
 
 
+def test_that_error_is_raised_when_a_file_has_been_staged_multiple_times(test_files):
```

Comment:
Yes, and I also realized that now that we are preventing files from being staged more than once, that outcome is occurring after the first successful execution of `stage` in a test. I.e. we are not testing what we used to be testing. I will add a helper function to the tests that clears out the `dcqc-staged-*` directories after each test calling `stage` so that each one is testing what it is meant to.